tsinfer/formats.py

```diff
@@ -2225,6 +2215,256 @@ def populations(self):
             yield Population(j, metadata=metadata)
 
 
+class SgkitSampleData(SampleData):
+
+    FORMAT_NAME = "tsinfer-sgkit-sample-data"
+    FORMAT_VERSION = (0, 1)
+
+    def __init__(self, path):
+        self.path = path
+        self.data = zarr.open(path, mode="r")
+        self._num_sites, self._num_individuals, self.ploidy = self.data[
```

Comment:
Maybe
```
genotypes_arr = self.data["call_genotypes"]
self._num_sites, self._num_individuals, self.ploidy = genotypes_arr.shape
```

Currently line breaking is a bit eye hurty

Reply:
https://github.com/tskit-dev/tsinfer/pull/748/commits/0ee3018cefa4d1ce2721b0b29ce7bc8006a22766